src/components/shared/form_elements/TextFormInput.vue

```diff
@@ -12,6 +12,8 @@
 			:placeholder="placeholder"
 			:disabled="disabled"
 			:required="required"
+			:aria-errormessage="ariaErrorMessage"
```

Comment:
I think some of the screen readers don't support `aria-errormessage` so we might have to change this to `aria-describedby`. We have [another ticket](https://phabricator.wikimedia.org/T361811) for that already so this isn't a blocker here.